test/modules/MetricsValidator/docker/linux/amd64/Dockerfile

```diff
@@ -1,6 +1,8 @@
 ﻿ARG base_tag=3.1.18-alpine3.13
-FROM mcr.microsoft.com/dotnet/runtime:${base_tag}
+# base dockerfile is located at \edge-util\docker\linux\amd64
+FROM azureiotedge/azureiotedge-runtime-base:1.2-linux-amd64 as builder
```

Comment:
You're not using `builder` anywhere in the Dockerfile...can it be removed?

Reply:
Ah I had a misunderstanding and thought there was more going on here. Copied all linux dockerfiles from the TRC, which uses the base image we want. But for amd64 we shouldn't copy the pattern directly because we don't need rocksdb. I think the arm32/64 changes are fine.